docs/source/common_issues.rst

```diff
@@ -85,6 +85,18 @@ flagged as an error.
   clarity about the latter use ``--follow-imports=error``.  You can
   read up about these and other useful flags in :ref:`command-line`.
 
+- **A function annotated as returning non-optional type but returns ``None``
```

Comment:
Hm, perhaps this should be:
>annotated as returning **a** non-optional type returns ``None``...

Reply:
You can make this small edit on the web yourself and then merge. No need to wait for OP.